example/app/storage/page.tsx

```diff
@@ -26,50 +30,47 @@ export default function StoragePage() {
             if (file) {
               upload.mutate({
                 bucketId: 'test',
-                fileId: 'test',
+                fileId: ID.unique(),
```

Comment:
Can you change all of the mutate hooks in storage service(wherever appropriate) to make things that required id for creation an optional argument?

Reply:
Sure!

Reply:
I believe the only one that should be updated was the upload. I've updated it and the example now.